packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -192,6 +192,10 @@ export function getElementName(
     if (name != null) {
       return name;
     }
+    const label = nextElement.getAttribute('aria-label');
```

Comment:
3 questions:
1- what about the `aria-labledby` support? We should detect that and find the corresponding element. 
2- should we give this more priority than the innerText? 
3- how do we communicate to outside of this function that this value is definitely not a UGC?

Reply:
I think the source would be useful for debugging as well so we know if it's `innerText`, `aria-label` or other source of the text,  potentially stuff this in metadata.

Maybe this function can return a result object `{text: string, source: 'innerText' | 'aria-label' | ...}`